src/pocketmine/entity/Entity.php

```diff
@@ -610,6 +610,10 @@ public function __construct(Level $level, CompoundTag $nbt){
 
 		$this->fallDistance = $this->namedtag->getFloat("FallDistance", 0.0);
 
+		if(!isset($this->width) or !isset($this->height)){
+			throw new \RuntimeException("Entity " . static::class . " must declare height and width properties and must be a float.");
```

Comment:
Now this is confusing. My entity must be a float?
The core of the problem is that a custom entity class hasn't declared height/width. Stick to that.

Reply:
Thank you for your review.
I removed "and must be a float".
```
RuntimeException: "Entity test\testEntity must declare height and width properties." (EXCEPTION) in "pmsrc/src/pocketmine/entity/Entity" at line 614
```